roles/cobbler/templates/snippets/cephlab_packages_rhel

```diff
@@ -1,3 +1,8 @@
 ## {{ ansible_managed }}
+#set os_version = $getVar('os_version','')
```

Comment:
I assume this var is set in cobbler somewhere?  Some docs explaining where this comes from might be helpful.